src/contracts/helpers.test.ts

```diff
@@ -32,11 +32,11 @@ describe('calculateAnnualInflationRate and friends', () => {
       expect(formatApi3(calculateAnnualMintedTokens(totalStake, apy))).toBe('55,284,457.03');
     });
 
-    test('calculateAnnualInflationRate', () => {
+    test('calculateAnnualTotalSupplyGrowth', () => {
       const apy = calculateApy(currentApr);
       const mintedTokens = calculateAnnualMintedTokens(totalStake, apy);
 
-      expect(calculateAnnualInflationRate(mintedTokens, API3_TOKEN_SUPPLY)).toBe(35.602054);
+      expect(calculateAnnualTotalSupplyGrowth(mintedTokens, API3_TOKEN_SUPPLY)).toBe(55.284456999999996);
```

Comment:
Does the numbers in the "corrected" tests make sense?

Reply:
Yeah